libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/resources/testProject/kaptIncrementalWithParceler/mylibrary/src/main/java/com/example/lib/ExampleParcel.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2011-2015 John Ericksen
```

Comment:
Could you just mention project from where sample was obtained and remove copyright?
Otherwise there are another TC fail:
```
  junit.framework.AssertionFailedError: The following files contain third-party copyrights and no license information. Please update license/README.md accordingly:
  libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/resources/testProject/kaptIncrementalWithParceler/mylibrary/src/main/java/com/example/lib/ExampleParcel.java:  * Copyright 2011-2015 John Ericksen
  junit.framework.AssertionFailedError: The following files contain third-party copyrights and no license information. Please update license/README.md accordingly:
  libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/resources/testProject/kaptIncrementalWithParceler/mylibrary/src/main/java/com/example/lib/ExampleParcel.java:  * Copyright 2011-2015 John Ericksen
```

Reply:
Done in  583dae2.